Sort/KthLargestElementInAStream.swift

```diff
@@ -0,0 +1,23 @@
+final class KthLargest {
```

Comment:
Could you add the question link and primary idea to solve this question? You can reference other Swift files for details.